CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Added
+- New spatial interpolation method to `UVBeam.interp` that improves that linear interpolation speed of `_interp_az_za_rect_spline`
```

Comment:
Maybe update this after the refactoring, it's a little weirdly worded now and give the new name to be passed to the `interp` method (`"az_za_map_coordinates"`).